lib/couchbase-orm/encrypt.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true, encoding: ASCII-8BIT
+
+module CouchbaseOrm
+    module Encrypt
+        def encode_encrypted_attributes
```

Comment:
I think it's overhead to parse all document to encode/decode document every time.
I preferred to use lazy way. 
At begin, all field is encrypted. 
When app want to read to attribute, model will decode the value and cache it.
When app want to write an attribute, model will encode value and cache value not encoded.

Reply:
Warning to separate 3 things : cypher - base 64 encoding - couchbase encoding

- cypher : the heaviest in term of computation, not handled by the library, in fact in docto it's handled by the front and cblite but not by the ruby monolith. So no performance problem
- base 64 encoding : I will remove it as it's not useable, a bit of useless computation removed.
- the couchbase format, no performance problem, it's tiny behind json stuff

What do you think ?